vlib/v/gen/c/testdata/const_init_or_block.vv

```diff
@@ -0,0 +1,9 @@
+// vtest vflags: -no-skip-unused
+fn t1() ?bool {
+	return true
+}
+
+type FooBar = bool
+
+const bar = FooBar(t1() or { panic(err) })
+const barz = FooBar(t1() or { panic(err) })
```

Comment:
That passes on master as well.
Is there a more specific case, that fails on master, but passes here?

Reply:
Also, can we print the constants, so that they are used, and to avoid the need for the vflags ?

If that is possible, the whole test can become a _test.v file, which is less brittle to changes in the output of cgen.

Reply:
I could not reproduce the failure from the original issue 🤔.

Reply:
It requires defining const with option unwrapping on different modules/files.